Content.Client/Computer/ComputerVisualizer.cs

```diff
@@ -43,12 +43,12 @@ public override void OnChangeData(AppearanceComponent component)
 
             if (!IoCManager.Resolve<IEntityManager>().TryGetComponent<SpriteComponent>(component.Owner, out var sprite)) return;
 
-            if (!component.TryGetData(ComputerVisuals.Powered, out bool powered))
+            if (!component.TryGetData<bool>(ComputerVisuals.Powered, out var powered))
```

Comment:
you can leave this file unchanged since another PR makes this a generic visualizer and avoiding conflicts is nice